pom.xml

```diff
@@ -35,6 +35,11 @@
       <artifactId>metrics-core</artifactId>
       <version>3.1.2</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-configuration2</artifactId>
```

Comment:
Can this dep be shaded?

Reply:
Yeah, but If it hasn't compatibility problems, it maybe no need to be shaded, otherwise, should all dependencys be shaded?

Reply:
I'm afraid so. The deps for non-testing code generally need to be shaded in case of dep conflict.
But given the apache.common.configuration2 is not much used here, we can leave it unshaded for now.